docs/skill_json.md

```diff
@@ -38,8 +41,11 @@ This file contains metadata about the skill and all info about how to install an
 On the JSON file:
 
 * `skill_id` refers to the unique skill identifier, this is defined in `setup.py` and by convention is usually `repo.author` (lower-cased)
-* `source` git url to download the source code, a skill can be installed from source with `pip install git+{source}`
+* `source` Optional git url to download the source code, a skill can be installed from source with `pip install git+{source}`
 * `package_name` the package name of the skill, if the skill is on pypi it can be installed with `pip install {package_name}`
+* `pip_spec` Optional [PEP 508](https://peps.python.org/pep-0508/#specification) install spec to install the skill. `pip install pip_spec`
+* `license` Optional [SPDX License ID](https://spdx.org/licenses/)
+* `author` Optional string author name. Overrides any name extracted from `skill_id` or `source` URL
 * `extra_plugins` list of python requirements that are not direct dependencies of the skill but should be installed in a companion OVOS service
 * `name` is a human-readable name for the skill to be used in store listings, UI, etc.
 * `description` is a human-readable short description of the skill. This should be limited to one or two sentences max
```

Comment:
name, extra_plugins and decription should also be flagged as optional for consistency, and a note added about where default values are extracted from (skill_id)